Nos/Views/AppView.swift

```diff
@@ -147,8 +148,34 @@ struct AppView: View {
             UITabBar.appearance().unselectedItemTintColor = .secondaryTxt
             UITabBar.appearance().tintColor = .primaryTxt
         }
+        .sheet(isPresented: $showNIP05Wizard) {
+            CreateUsernameWizard(isPresented: $showNIP05Wizard)
+        }
+        .task(presentNIP05SheetIfNeeded)
         .accentColor(.primaryTxt)
     }
+
+    @Sendable private func presentNIP05SheetIfNeeded() async {
+        // Sleep for half a second
+        try? await Task.sleep(nanoseconds: 500_000_000)
+        guard let author = currentUser.author, let npub = author.npubString else {
+            return
+        }
+        // We store the npub for the user we last presented the sheet for so that
+        // the behavior resets if the user creates a new account
+        let key = "AppView_presentedNIP05SheetFor"
```

Comment:
I don't know how much it matters that `AppView` presented it, so I might suggest a couple tweaks:
```suggestion
        let key = "didPresentNIP05SheetForNpub"
```